docker-for-windows/docker-toolbox.md

```diff
@@ -1,17 +1,60 @@
 ---
 description: Docker Desktop for Windows and Docker Toolbox
 keywords: windows, alpha, beta, toolbox, docker-machine, tutorial
```

Comment:
can you add a `hide_from_sitemap` so that it doesn't get indexed by Google?

```suggestion
keywords: windows, alpha, beta, toolbox, docker-machine, tutorial
hide_from_sitemap: true
```